types/bluebird/index.d.ts

```diff
@@ -755,7 +755,7 @@ declare class Bluebird<R> implements PromiseLike<R>, Bluebird.Inspection<R> {
   static promisify<T, A1, A2, A3>(func: (arg1: A1, arg2: A2, arg3: A3, callback: (err: any, result?: T) => void) => void, options?: Bluebird.PromisifyOptions): (arg1: A1, arg2: A2, arg3: A3) => Bluebird<T>;
   static promisify<T, A1, A2, A3, A4>(func: (arg1: A1, arg2: A2, arg3: A3, arg4: A4, callback: (err: any, result?: T) => void) => void, options?: Bluebird.PromisifyOptions): (arg1: A1, arg2: A2, arg3: A3, arg4: A4) => Bluebird<T>;
   static promisify<T, A1, A2, A3, A4, A5>(func: (arg1: A1, arg2: A2, arg3: A3, arg4: A4, arg5: A5, callback: (err: any, result?: T) => void) => void, options?: Bluebird.PromisifyOptions): (arg1: A1, arg2: A2, arg3: A3, arg4: A4, arg5: A5) => Bluebird<T>;
-  static promisify(nodeFunction: Function, options?: Bluebird.PromisifyOptions): Function;
+  static promisify(nodeFunction: (...args: any[]) => void, options?: Bluebird.PromisifyOptions): (...args: any[]) => Bluebird<any>;
```

Comment:
I believe the `nodeFunction` should be `(...args: any[]) => any` as the return value is ignored, right?

Reply:
thank review!
According to typescript playbook, [callback should be of type to return void](http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#callback-types).
Also, I checked the source code of bluebird and confirmed that the [return value of callback is discarded](https://github.com/petkaantonov/bluebird/blob/master/src/promisify.js#L233).
Based on the above, I have created PR to return void.
It will be helpful if you can judge from these.

Reply:
@segayuu Ah, I only saw your comment just now. Sorry for the confusion - I'll approve it now. 🙂